+++ lib/command/init.rb
@@ -128,7 +128,7 @@
       puts
       # ファイルコピー
       src = ["AozoraEpub3.ini", "vertical_font.css"]
-      dst = ["AozoraEpub3.ini", "template/OPS/css_custom/vertical_font.css"]
+      dst = ["AozoraEpub3.ini", "template/item/style_custom/font.css"]
       puts "(次のファイルをコピーor上書きしました)"
       src.size.times do |i|
         src_full_path = File.join(Narou.preset_dir, src[i])
+++ lib/novelconverter.rb
@@ -86,7 +86,7 @@
   end

   DAKUTEN_FROM = ["vertical_font_with_dakuten.css", "DMincho.ttf"]
-  DAKUTEN_TO = ["template/OPS/css_custom/vertical_font.css", "template/OPS/fonts/DMincho.ttf"]
+  DAKUTEN_TO = ["template/item/style_custom/font.css", "template/item/fonts/DMincho.ttf"]
   DAKUTEN_ERB = [true, false]

   def self.activate_dakuten_font_files
@@ -100,6 +100,7 @@
       if DAKUTEN_ERB[i]
         Helper.erb_copy(src, dst, binding)
       else
+        FileUtils.mkdir_p(File.dirname(dst))
         FileUtils.copy(src, dst)
       end
     end
+++ preset/vertical_font.css
@@ -1,12 +1,21 @@
 @charset "utf-8";
 @namespace "http://www.w3.org/1999/xhtml";

+/** 横書きフォント */
+.hltr body {
+font-family: "ＭＳ 明朝", "MS Mincho", "ヒラギノ明朝 ProN W3", "HiraMinProN-W3", serif, sans-serif;
+}
+/** 縦書きフォント */
+.vrtl body {
+font-family: "@ＭＳ 明朝", "@MS Mincho", "ヒラギノ明朝 ProN W3", "HiraMinProN-W3", serif, sans-serif;
+}
+
 body {
-       font-family: "@ＭＳ 明朝", "@MS Mincho", "ヒラギノ明朝 ProN W3", "HiraMinProN-W3", serif, sans-serif;
+       /* font-family: "@ＭＳ 明朝", "@MS Mincho", "ヒラギノ明朝 ProN W3", "HiraMinProN-W3", serif, sans-serif; */
        line-height: <%= line_height %>em !important;
 }

-.gtc, .b {
+.vrtl .gtc, .vrtl .b {
        font-family: '@ＭＳ ゴシック','@MS Gothic',sans-serif !important;
 }

+++ preset/vertical_font_with_dakuten.css
@@ -1,12 +1,21 @@
 @charset "utf-8";
 @namespace "http://www.w3.org/1999/xhtml";

+/** 横書きフォント */
+.hltr body {
+font-family: "ＭＳ 明朝", "MS Mincho", "ヒラギノ明朝 ProN W3", "HiraMinProN-W3", serif, sans-serif;
+}
+/** 縦書きフォント */
+.vrtl body {
+font-family: "@ＭＳ 明朝", "@MS Mincho", "ヒラギノ明朝 ProN W3", "HiraMinProN-W3", serif, sans-serif;
+}
+
 body {
-       font-family: "@ＭＳ 明朝", "@MS Mincho", "ヒラギノ明朝 ProN W3", "HiraMinProN-W3", serif, sans-serif;
+       /* font-family: "@ＭＳ 明朝", "@MS Mincho", "ヒラギノ明朝 ProN W3", "HiraMinProN-W3", serif, sans-serif; */
        line-height: <%= line_height %>em !important;
 }

-.gtc, .b {
+.vrtl .gtc, .vrtl .b {
        font-family: '@ＭＳ ゴシック','@MS Gothic',sans-serif !important;
 }

